COMETwebapp/Components/ParameterTable.razor

```diff
@@ -52,25 +52,33 @@ Copyright (c) 2022 RHEA System S.A.
     /// </summary>
     protected override void OnInitialized()
     {
-        foreach(var element in ISessionAnchor.OpenIteration.Element)
+        var ListElements = ISessionAnchor.OpenIteration.Element;
+        if (ListElements.Count != 0)
         {
-            foreach(var parameter in element.Parameter)
+            foreach (var element in ListElements)
             {
-                foreach (var valueset in parameter.ValueSet) {
-                    Parameters.Add(new BuiltParameter()
+                var ListParameters = element.Parameter;
```

Comment:
please do not use the word `list` in variable names, `parameters` is good enough